### DISCUSSION TITLE
Is Chrome no longer supported to download videos?

### DISCUSSION THREAD

**Discussion by neroconsulting** [aclap-dev/video-downloadhelper#1016] [Help and Q&A]
### Your question

I just purchased a license but every video in Chrome gives me the message:

Unfortunately, the Chrome Web Store does not allow extensions for downloading YouTube videos so we could not include this feature in the Chrome version of the extension.

You can use Video DownloadHelper to download YouTube videos on the Firefox version.

### Addon & CoApp version and details

Unfortunately, the Chrome Web Store does not allow extensions for downloading YouTube videos so we could not include this feature in the Chrome version of the extension.

You can use Video DownloadHelper to download YouTube videos on the Firefox version.

**Comment by jcfrog** [comment]
Hi,
Chrome never allowed Youtube downloads.
If you want to change your premium (if it was purchased less than a month ago), send us an email to licenses@downloadhelper.net so that you can purchase Firefox or Edge premium if you want to.

**Reply by sergeevabc** [reply]
Use Edge version of VideoDownload Helper ([store](https://microsoftedge.microsoft.com/addons/detail/video-downloadhelper/jmkaglaafmhbcpleggkmaliipiilhldn), [crx](https://edge.microsoft.com/extensionwebstorebase/v1/crx?x=id%3Djmkaglaafmhbcpleggkmaliipiilhldn%26installsource%3Dondemand&response=redirect)) without the mentioned limitation.
It's not the fastest solution for YouTube (compared to [yt-dlp](https://github.com/nicolaasjan/yt-dlp), for example), but it works.
![](https://github.com/user-attachments/assets/6fdba3af-494e-4923-a91e-2eab2960b5e8)
You may also need to edit `chrome.json` (for Chromium-based browser) or `microsoft.json` (for Edge browser) in the CoApp folder (e.g. `C:\Program Files\DownloadHelper CoApp`) by adding the extension ID to the `allowed_origins` list (e.g. `"chrome-extension://bhnicdfkkjekngfbijpgfcndlfmepcdh/",`).
